packages/server/src/util/indicators/index.ts

```diff
@@ -0,0 +1,140 @@
+import { LiquidityBound } from '@sommelier/shared-types';
+import { DexTrade } from 'services/bitquery/generated-types';
+
+export function getAllIndicators(marketData: DexTrade[]): { [indicatorName: string]: LiquidityBound } {
+    return {
+        bollingerSMANormalBand: getSMABollingerBands(marketData),
+        bollingerSMAWideBand: getSMABollingerBands(marketData, 2.5),
+        bollingerEMANormalBand: getEMABollingerBands(marketData),
+        bollingerEMAWideBand: getEMABollingerBands(marketData, 2.5),
+        rsi: getRSILevels(marketData)
+    }
+}
+
+export function getEMABollingerBands(marketData: DexTrade[], numStdDevs = 2): LiquidityBound {
+    const sampleData = marketData[0];
+    const ma = getEMA(marketData);
+    const stddev = getStandardDeviation(marketData);
+    const boundTerm = numStdDevs * stddev;
+
+    return {
+        baseTokenId: sampleData.baseCurrency.address,
+        quoteTokenId: sampleData.quoteCurrency.address,
+        indicatorName: 'BollingerBand',
```

Comment:
Do we need to differentiate EMA vs SMA in the indicator name?

Reply:
I did this in such a way where I override the `indicatorName` in `getAllIndicators` - I figured the semantics about `normal/wide` band made more sense there, and it would be weird to name it here (unless we change the name to e.g. `2StdDevBand`).

Let me know if you think it's dirty

Reply:
Got it